helper/raftutil/state.go

```diff
@@ -160,3 +161,62 @@ func commandName(mt structs.MessageType) string {
 
 	return fmt.Sprintf("%v", mt)
 }
+
+// FindRaftFile finds raft.db and returns path
+func FindRaftFile(p string) (raftpath string, err error) {
```

Comment:
This method only delegates to `FindRaftInPath`.  I'd just remove it and rename the other function instead.

Reply:
This was heading toward a generic function to handle finding raft.db and state.bin.  I completed the refactor and it should make more sense now.